contracts/feeds/customfeeds/PriceFeedCurvestETH.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity ^0.8.0;
```

Comment:
I prefer to have all constants uppercase

Reply:
I changed all constants names to uppercase for both this PR and stMATIC PR.